### PR TITLE
cifuzz,cflite: set mmap_rnd_bits to 28

### PR DESCRIPTION
**Pull request by evverx**
to get MSan jobs to work with the latest Ubuntu images.

https://github.com/google/sanitizers/issues/1614
https://github.com/actions/runner-images/issues/9491

### CODE REVIEW

File: .github/workflows/cflite_pr.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ jobs:
       matrix:
         sanitizer: [address, undefined, memory]
     steps:
+    - run: sudo sysctl -w vm.mmap_rnd_bits=28
```
Comment (yuwata) [inline .github/workflows/cflite_pr.yml:26]: Could you add a short comment about this?

File: .github/workflows/cifuzz.yml [hunk 2]
```diff
@@ -39,6 +39,7 @@ jobs:
       security-events: write
 
     steps:
+      - run: sudo sysctl -w vm.mmap_rnd_bits=28
```
Comment (yuwata) [inline .github/workflows/cifuzz.yml:42]: Also here.

**Review by yuwata — APPROVED** [review]

File: .github/workflows/cflite_pr.yml (hunk 1 above)
Comment (evverx) [reply .github/workflows/cflite_pr.yml:26]: GitHub should roll back that change next week so I was planning to just open a PR reverting this (and b7c7498de814b1e9825b43c28e307a7f0af8ffd2).

Comment (yuwata) [reply .github/workflows/cflite_pr.yml:26]: OK, then let's merge this as is.

Comment (evverx) [reply .github/workflows/cflite_pr.yml:26]: I opened https://github.com/systemd/systemd/pull/31803

**Pull request merged**